decidim-templates/app/views/decidim/templates/admin/questionnaire_templates/_preview.html.erb

```diff
@@ -47,10 +47,11 @@
 
                 <% if step_index + 1 == @preview_form.total_steps %>
                   <div class="row column tos-agreement">
-                    <%= form.check_box :tos_agreement, label: t(".tos_agreement"), id: "questionnaire_tos_agreement", disabled: false %>
-                    <div class="help-text">
-                      <%= decidim_sanitize translated_attribute questionnaire.tos %>
-                    </div>
+                    <%= form.check_box :tos_agreement,
+                      label: t(".tos_agreement"),
+                      id: "questionnaire_tos_agreement",
+                      disabled: false,
+                      help_text: decidim_sanitize(translated_attribute(questionnaire.tos)) %>
```

Comment:
This text is being renedered as: 
```
<span class="help-text"><p>Porro est illo. Repellat commodi deserunt.</p></span>
```

Reply:
Ok, I undo this change at c690958a399b4488d0f9930ed4e38adcc124a67d